src/cli.ts

```diff
@@ -85,6 +85,7 @@ program
     '-o, --out-file [file-path]',
     'Path to a file that fqm-execution will write the calculation results to (default: output.json)'
   )
+  .option('--cache-elm', 'Whether or not to cache the ELM JSONs and associated information from calculation', false)
```

Comment:
Question: do we want an option for clearElmJsonsCache?

Reply:
I left out this option because I think the cache should only work when fqm-execution is run as a package as part of a running program, like in fqm-testify. The cli commands will only be run between isolated runs of fqm-execution, so I think the cache will reset regardless. @mgramigna am I right about this? Happy to add a clear cache option if not.

Reply:
Ah that makes sense now!!

Reply:
correct

Reply:
I hate to throw a wrench in this PR review. But there isn't a need for the `--cache-elm` option in the CLI at all since it's just an in memory cache. So it will be built fresh every single run.

Reply:
Lol that's a great point. Will remove